### DISCUSSION TITLE
Nominate @arjun024 as a maintainer

### DISCUSSION THREAD

**Discussion by sophiewigmore** [paketo-buildpacks/feedback#184] [PHP Team]
I'd like to nominate @arjun024 as a PHP maintainer. He was previously a Paketo PHP maintainer and has worked on the buildpacks a lot in the past, for example, he wrote the RFC for the [PHP buildpack rearchitecture](https://github.com/paketo-buildpacks/rfcs/blob/main/text/php/0001-restructure.md). He is now available again to work on the project, so I'd like to re-nominate him as a maintainer

@paketo-buildpacks/php-maintainers can I get a 👍 or 👎 ?

**Comment by sophiewigmore** [comment]
With 2/2 votes, this nomination passes. Congrats @arjun024. @paketo-buildpacks/steering-committee Could you please add Arjun to the PHP maintainers team, please?

**Reply by ForestEckhardt** [reply]
Sure thing could you update the TEAMS.md in the community repo?

**Reply by sophiewigmore** [reply]
https://github.com/paketo-buildpacks/community/pull/146
